docs/content/users/quickstart.md

```diff
@@ -250,8 +249,7 @@ While the generic `php` project type is [ready to go](./project.md) with any CMS
             mkdir my-statamic-app
             cd my-statamic-app
             ddev config --project-type=laravel --docroot=public --create-docroot
-            ddev composer create --prefer-dist --no-install --no-scripts statamic/statamic
-            ddev composer install
+            ddev composer create --prefer-dist statamic/statamic
             ddev exec "php artisan key:generate"
             ddev launch
```

Comment:
```suggestion
            ddev exec "php artisan key:generate"
            ddev exec "php please make:user" # Add a super user
            ddev launch
```

Reply:
I was just uncertain if it would be the better choice to add the comment `# Add a super user` to a dedicated line before the please make:user command or if it is the better choice to make the comment in line like i did in the suggestion